data/tabular/freesolv/meta.yaml

```diff
@@ -90,15 +90,15 @@ templates:
       \ {#Understood|Got it|Ok!}, this {SMILES__description} represents a molecule that has a {GAFF__names__noun} of {GAFF#} {GAFF__units}: {SMILES#}"
     - The {exp_value__names__noun} of the molecule with the {SMILES__description} {SMILES#} is:<EOI> {exp_value#} {exp_value__units}
     - The {exp_value__names__noun} of the {SMILES__description} {SMILES#} is:<EOI> {exp_value#} {exp_value__units}
-    - The {exp_value__names__noun} of the molecule {SMILES__description} {SMILES#} is:<EOI> {exp_value#} {exp_value__units}
+    - The {exp_value__names__noun} of the molecule with {SMILES__description} {SMILES#} is:<EOI> {exp_value#} {exp_value__units}
```

Comment:
```suggestion
    - The {exp_value__names__noun} of the molecule with the {SMILES__description} {SMILES#} is:<EOI> {exp_value#} {exp_value__units}
```

Reply:
or remove `with`

Reply:
well, I have this question for most changes here :)